doc/hooks-config.rst

```diff
@@ -84,6 +88,10 @@ translations to British English and Simplified Chinese:
                 "icons": ["Adwaita"],
                 "themes": ["Adwaita"],
                 "languages": ["en_GB", "zh_CN"],
+                "module-versions": {
+                    "Gtk": "3.0",
+                    "GtkSource": "4",
+                },
```

Comment:
This rather insinuates that any Gtk module version can be configured this way. Can we have a temporary
```rst
.. note:: Currently only the version of ``GtkSource`` is supported.
```
until we've gone through all the other modules too (unless you already plan to do that soon)?